### PR TITLE
Administration endpoints and dependancies

### PR DESCRIPTION
**Pull request by Berardpi**
* Adds endpoints for mno-hub (sync status, launch sync, disconnect account)
* Adds a dependancies endpoint (dependancies of mno-components)
* Adds a method to get the current connec! version of the tenant

@BrunoChauvet Can you take a quick look? Especially the json response for the endpoints to check that it fits what we need in MnoHub

### CODE REVIEW

**Comment by Berardpi** [comment]
@BrunoChauvet I've also added a commit for a batched first synchronization as discussed on slack. I don't have a good way right now to store the state to be able to continue in case of error... if you have any miracle idea

File: app/controllers/maestrano/synchronizations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  end
+
+  def create
+    uid = params[:group_id]
+    opts = params[:opts] || {}
+    organization = Maestrano::Connector::Rails::Organization.find_by_uid(uid)
+    return render json: { errors: {message: "Organization not found", code: 404} }, status: :not_found unless organization
+
+    Maestrano::Connector::Rails::SynchronizationJob.perform_later(organization, opts.with_indifferent_access)
+    head 201, content_type: "application/json"
```
Comment (BrunoChauvet) [inline app/controllers/maestrano/synchronizations_controller.rb]: I think you ca do
```ruby
head :created
```

File: app/controllers/maestrano/synchronizations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  end
+
+  def destroy
+    uid = params[:id]
+    organization = Maestrano::Connector::Rails::Organization.find_by_uid(uid)
+    return render json: { errors: {message: "Organization not found", code: 404} }, status: :not_found unless organization
+
+    organization.update(sync_enabled: false)
+    head 200, content_type: "application/json"
```
Comment (BrunoChauvet) [inline app/controllers/maestrano/synchronizations_controller.rb]:
```ruby
head :ok
```

Comment (BrunoChauvet) [inline app/controllers/maestrano/synchronizations_controller.rb]: Thinking about it, shouldn't we have an endpoint `toggle`?
```ruby
organization.update(sync_enabled: !organization.sync_enabled)
```

**Comment by Berardpi** [comment]
@BrunoChauvet I've ETE it, almost worked.
I've also change the endpoint for a toggle one, good idea 👍

**Pull request closed without merging**